task160_FinalTask/src/test/java/automationpractice/com/pages/WishlistPage.java

```diff
@@ -0,0 +1,68 @@
+package automationpractice.com.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.Random;
+
+public class WishlistPage extends BasePage {
+    @FindBy(className = "table")
+    private WebElement WISHLIST_TABLE;
+    @FindBy(css = "li.clearfix>a")
+    private List<WebElement> LIST_OF_SOME_PRODUCTS;
+    @FindBy(css = "tr>td:first-child>a")
+    private WebElement MY_WISHLIST_LINK;
+    @FindBy(css = "ul.wlp_bought_list>li:first-child div.product_image>a")
+    private WebElement MY_FIRST_ITEM_IN_MY_WISHLIST;
+    @FindBy(css = ".product_image")
+    private WebElement MY_FIRST_ITEM_IN_MY_WISHLIST_TEST;
+    @FindBy(css = "tbody>tr")
+    private List<WebElement> LIST_OF_WISHLISTS;
+    @FindBy(css = "tbody>tr:first-child a.icon")
+    private WebElement FIRST_DELETE_ICON;
+    @FindBy(id = "name")
+    private WebElement NEW_WISHLIST_NAME_INPUT;
+    @FindBy(id = "submitWishlist")
+    private WebElement SAVE_WISHLIST_BUTTON;
+
+    public WishlistPage() {
+
+    }
```

Comment:
I think you can remove it

Reply:
I'm actually don't want to remove it, I'm checking if I on the expected page or not. 

This check was missing, i've added it.